test/db/tools/rabin2

```diff
@@ -1045,6 +1045,15 @@ EXPECT=<<EOF
 EOF
 RUN
 
+NAME=rabin2 -Pj
+FILE=bins/pdb/minimal.pdb
+CMDS=!rabin2 -Pj ${R2_FILE}
```

Comment:
append `~j` to get the JSON indent output, its easier to read and check the diff in case the output changes.